2020-08-27/cpuinfo

```diff
@@ -0,0 +1 @@
+cp /proc/cpuinfo .
```

Comment:
Can you type
  * `cp /proc/cpuinfo .`
  * `cp /proc/meminfo .`
in this directory and in 2020-09-10 and commit the result so that we know by which type of machine the benchmark have been run.